gammacat/input.py

```diff
@@ -216,6 +216,47 @@ def validate(self):
         log.info('Validating YAML files in `input/sources`')
         [_.validate() for _ in self.data]
 
+class InputInfo:
+    """All basic info about the data of a single publication.
+    """
+
+    schema = load_yaml(gammacat_info.base_dir / 'input/schemas/dataset_info.schema.yaml')
+
+    def __init__(self, data, path):
+        self.data = data
+        self.path = path
+
+    @classmethod
+    def read(cls, path):
+        path = Path(path)
+        data = load_yaml(path)
+        return cls(data=data, path=path)
+
+    def validate(self):
+        validate_schema(path=self.path, data=self.data, schema=self.schema)
+
+class InputInfoCollection:
```

Comment:
This class is a collection of all info-files which are stored in the input folder. 
The only thing that it can be doing is to go through all files and validate each one using `InputInfo.validate()`. But - same as above - there may be more functionality in the future.